packages/implementations-lib/src/transfers/mongo_transfers_repo.ts

```diff
@@ -298,41 +330,42 @@ export class MongoTransfersRepo implements ITransfersRepository {
 	}
 
 
-	async getSearchKeywords():Promise<{fieldName:string, distinctTerms:string[]}[]>{
-        const retObj:{fieldName:string, distinctTerms:string[]}[] = [];
+	async getSearchKeywords(): Promise<{ fieldName: string, distinctTerms: string[] }[]> {
+		const retObj: { fieldName: string, distinctTerms: string[] }[] = [];
 
-        try {
-            const result = this.transfers
-                .aggregate([
-					{$group: { "_id": { transferState: "$transferState", currencyCode: "$currencyCode" } } }
+		try {
+			const result = this.transfers
+				.aggregate([
+					{ $group: { "_id": { transferState: "$transferState", currencyCode: "$currencyCode" } } }
 				]);
 
-			const state:{fieldName:string, distinctTerms:string[]} = {
+			const state: { fieldName: string, distinctTerms: string[] } = {
 				fieldName: "state",
 				distinctTerms: []
 			};
 
-			const currency:{fieldName:string, distinctTerms:string[]} = {
+			const currency: { fieldName: string, distinctTerms: string[] } = {
 				fieldName: "currency",
 				distinctTerms: []
 			};
 
 			for await (const term of result) {
```

Comment:
From the comment above, if you look at this method you'll notice how we make the `distinct` of the terms that are found for `state` and `currencyCode`. If an input is supposed to be a list, then it needs to exist here.